packages/capabilities/src/space.js

```diff
@@ -63,3 +63,32 @@ export const allocate = capability({
     }
   },
 })
+
+/**
+ * The capability grants permission for all content serve operations that fall under the "space/content/serve" namespace.
+ * It can be derived from any of the `space/*` capability that has matching `with`.
+ */
+
+export const contentServe = capability({
+  can: 'space/content/serve/*',
+  with: SpaceDID,
+  derives: equalWith,
+})
+
+/**
+ * Capability can be invoked by an agent to record egress data for a given resource.
+ * It can be derived from any of the `space/content/serve/*` capability that has matching `with`.
+ */
+export const recordEgress = capability({
```

Comment:
...to echo the ability name?

```suggestion
export const egressRecord = capability({
```